packages/block-editor/src/components/block-full-height-alignment-toolbar/icon.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+
+export default (
```

Comment:
We have a WordPress/icons package. Would it make sense to add this icon there?

Reply:
I'd say so, yes.